src/main/resources/Mobs.json

```diff
@@ -1,7 +1,22 @@
-{
-  "Mob":{
```

Comment:
Keep object name, can be replaced with mobID or name to make file easier to navigate

Reply:
Same as above